src/documentation/ntfs2gtfs.md

```diff
@@ -0,0 +1,147 @@
+service_id | yes | trips.txt | service_id | 
+trip_id | yes | trips.txt | trip_id | 
+trip_headsign | no | trips.txt | | (1)
+trip_short_name | no | trips.txt | | (1)
+trip_desc | no | comments.txt, comment_links.txt | | The value of `comment_name` referenced by the `comment_id` having an `object_type` = `trip` and an `object_id` equal to the corresponding `trip_id`. In case of more than one comments linked to the same trip, the first one in alphabetical order is taken into account.
+direction_id | no | routes.txt | direction_type | `1` if the corresponding value is `forward`, `clockwise` or `inbound`. `0` otherwise.
+block_id | no | trips.txt | block_id | 
+shape_id | no | trips.txt | geometry_id | (link to the [shapes.txt](#shapes.txt) file)
+wheelchair_accessible | no | trip_properties.txt | wheelchair_accessible | The value of `wheelchair_accessible` referenced by the `trip_property_id` of this trip.
+bikes_allowed | no | trip_properties.txt | bike_accepted | The value of `bike_accepted` referenced by the `trip_property_id` of this trip.
+
+(1) If the value of the `physical_mode_id` for this trip is `LocalTrain`, `LongDistanceTrain`, `Metro`, `RapidTransit` or `Train`,
+- the field `trip_short_name` takes after the value of the `trip_headsing` field in the NTFS
+- the field `trip_headsign` contains the value of the `stop_name` linked to the stoppoint of this trip with the higher order in the trip (i.e. the name of the last stop of the trip).
+
+Otherwise,
+- the field `trip_short_name` is empty
+- the field `trip_headsign` contains
+  - the value of the `trip_headsing` field in the NTFS, if present
+  - otherwise, the value of the `stop_name` linked to the stoppoint of this trip with the higher order in the trip (i.e. the name of the last stop of the trip).
+
+### stop_times.txt
+
+GTFS field | Required | NTFS file | NTFS field | Note
+--- | --- | --- | --- | ---
+trip_id | yes | stop_times.txt | trip_id | (link to the [trips.txt](#trips.txt) file)
+arrival_time | yes | stop_times | arrival_time | 
+departure_time | yes | stop_times.txt | departure_time | 
+stop_id | yes | stop_times.txt | stop_id | (link to the [stops.txt](#stops.txt) file)
+stop_sequence | yes | stop_times.txt | stop_sequence |
+stop_headsign | no | stop_times.txt | stop_headsign | 
+pickup_type | no | stop_times.txt | pickup_type |
+drop_off_type | no | stop_times.txt | drop_off_type |
+stop_time_desc | no | comments.txt, comment_links.txt | | The value of `comment_name` referenced by the `comment_id` having an `object_type` = `stop_point`and an `object_id` equal to the corresponding `trip_id`. In case of more than one comments linked to the same stop, the first one in alphabetical order is taken into account.
+local_zone_id | no | stop_times.txt | local_zone_id |
+
+### calendar_dates.txt
+This file is the same as the NTFS calendar_dates.txt.
+
+### transfers.txt
+This file is generated only if the corresponding NTFS file is present and non empty.
```

Comment:
This rule should be a global one. For exemple, shapes.txt should not be generated if empty.